tests/integration_cases/test_membership_changes.rs

```diff
@@ -0,0 +1,1690 @@
+// Copyright 2019 PingCAP, Inc.
```

Comment:
I think it wasn't intended to have this added. It was part of #101 https://github.com/pingcap/raft-rs/pull/101/files#diff-e06fb4f47cfc0335fe0f7ec1d7268e2e